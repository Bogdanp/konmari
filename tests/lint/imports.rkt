#lang racket/base

(require (for-syntax racket/base
                     syntax/parse)
         racket/contract)
