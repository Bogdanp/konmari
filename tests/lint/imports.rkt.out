/imports.rkt:4:22:warning:prefer syntax/parse/pre if possible
/imports.rkt:5:10:warning:prefer racket/contract/base if possible
